config/initializers/action_dispatch.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.configure do
+  config.action_dispatch.default_headers.merge!(
+    'X-UA-Compatible' => 'IE=Edge'
+  )
+end
```

Comment:
secure_headers doesn't cover this IE-specific header, so adding this initializer here and merging it with [Action Dispatch's default headers](http://guides.rubyonrails.org/configuring.html#configuring-action-dispatch).